index.html

```diff
@@ -114,10 +114,17 @@ <h2 style="text-align:center;">Currently in use at</h2>
                     <a href="http://www.calpoly.edu"><li class="col"><img src="assets/calpoly.svg" class="greydout" alt="CalPoly" height="60" width="72"></li></a>
                     <a href="http://www.berkeley.edu"><li class="col"><img src="assets/Berkeley.svg" class="greydout" alt="Berkeley" height="60" width="147"></li></a>
                     <a href="http://www.sheffield.ac.uk/"><li class="col"><img src="assets/sheffield.svg" class="greydout" alt="Sheffield" height="60" width="137"></li></a>
+<<<<<<< HEAD
```

Comment:
Looks like you've committed some conflict markers. Can you remove these and resolve the conflicts?